integrations/tensorflow/e2e/bool_test.py

```diff
@@ -20,40 +20,40 @@
 
 
 class MathModule(tf.Module):
+
   @tf.function(input_signature=[tf.TensorSpec([4], tf.float32)])
   def greater_than(self, x):
     return x > 1.0
 
-  @tf.function(input_signature=[tf.TensorSpec([4], tf.bool),
-                                tf.TensorSpec([4], tf.bool)])
+  @tf.function(input_signature=[
+      tf.TensorSpec([4], tf.bool),
+      tf.TensorSpec([4], tf.bool)
+  ])
```

Comment:
It would be nice to unify the Python formatting between open source development and what Google does. (Pick one or the other and enforce here on GitHub). Same with markdown.